portfolio/src/main/webapp/index.html

```diff
@@ -5,8 +5,6 @@
     <title>Jordan's Site</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-
-    <script src = "https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.js"></script>
```

Comment:
Remove this by accident, or does it have some impact on your webpage?

Reply:
I was using a CDN in order to use Node.js functions but I have just requested access to Node.js